stubs/docutils/docutils/frontend.pyi

```diff
@@ -60,6 +61,14 @@ class OptionParser(optparse.OptionParser, SettingsSpec):
     default_error_encoding_error_handler: ClassVar[str]
     config_section: ClassVar[str]
     version_template: ClassVar[str]
+    def __init__(
+        self,
+        components: Tuple[Type[Parser]] = ...,
+        defaults: Dict[str, object] = ...,
```

Comment:
This can also be any `Mapping`

```suggestion
        defaults: Mapping[str, object] = ...,
```

(Also needs an import from `collections.abc`.)

Reply:
Done.